scripts/publish-starters.sh

```diff
@@ -27,8 +27,8 @@ for folder in $GLOB; do
   cp -r "$BASE/$folder/." .
 
   if [ "$IS_WORKSPACE" = null ]; then
-    rm -rf yarn.lock
-    yarn import # generate a new yarn.lock file based on package-lock.json
+    // we remove yarn.lock, our gatsby new command uses `yarn import` to create it.
+    rm -f yarn.lock
```

Comment:
not the hill I will die on, but this somewhat assumes that users will upgrade cli (their cli would use `yarn import` on their own). Would it cause problems if we kept `yarn.lock` around in our starters?

Reply:
Users will see warning from npm that both lock files were found, won't they?

Reply:
`gatsby new` deletes not used lock files (based on configured package manager), so the warning wouldn't show

Reply:
I believe it does blow up based on our testing. But it's a fair point that this is decoupled from CLI version. 🤔